server/src/test/java/io/crate/integrationtests/GroupByAggregateTest.java

```diff
@@ -582,6 +582,41 @@ public void testGroupByCountOnColumn() throws Exception {
         assertEquals(1L, response.rows()[3][2]);
     }
 
+    @Test
+    public void testGroupByCountOnObject() {
```

Comment:
Same here, please rename method to clarify the scenario

Reply:
renamed, `testCompareCountOnObjectHavingNotNullSubcolumnAndCountDirectlyOnNotNullSubcolumn`